### PR TITLE
Visualisation of the POI being analysed by MeiliLens

### PR DESCRIPTION
**Pull request by MarcelTorne**
In this PR I am bringing the feature of displaying in a different colour (violet for the moment, but open to proposals) the POI that MeiliLens has found.

### CODE REVIEW

**Comment by MarcelTorne** [comment]
> Great addition! All I see is code improvement. Unfortunately, I can't test the feature since I don't have an android phone but I can approve the PR as soon as another teammate confirms the feature works!

Thanks for the comment @ahmedc3ca ! If you still want to test the feature from the emulator on your laptop you can go in the `...`button on the emulator and then in `virtual sensors`and tweak the rotation so you'll see the colour changing.

File: app/src/main/java/com/github/epfl/meili/map/PoiRenderer.kt [hunk 1]
```diff
@@ -4,47 +4,87 @@ import android.content.Context
...
     companion object {
-        val REACHABLE_ICON = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_AZURE)
-        val VISITED_ICON = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_YELLOW)
-        val VISIBLE_ICON = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_GREEN)
-        val DEFAULT_ICON = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_RED)
+        val REACHABLE_ICON: BitmapDescriptor = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_AZURE)
+        val VISITED_ICON: BitmapDescriptor = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_YELLOW)
+        val VISIBLE_ICON: BitmapDescriptor = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_GREEN)
+        val DEFAULT_ICON: BitmapDescriptor = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_RED)
+        val MEILI_LENS_ICON: BitmapDescriptor = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_VIOLET)
```
[the remark this answers is not in the document]
Comment (MarcelTorne) [reply app/src/main/java/com/github/epfl/meili/map/PoiRenderer.kt:88]: I think these colours look great in dark mode but we definitely can discuss this and add this task to the backlog. Thanks for the suggestion and let's discuss it offline!

**Review by Bwananah — APPROVED** [review]
Works as intended. We should definitely fix the MeiliLens flickering issue if possible, but your feature implementation is great !

**Comment by ahmedc3ca** [comment]
Thanks for the suggestion didn't know this feature existed. It works as intended 👍

**Review by ahmedc3ca — APPROVED** [review]

**Pull request merged**